src/CommunityToolkit.Maui.Core/Primitives/Rating.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace CommunityToolkit.Maui.Core.Primitives;
+
+///<summary></summary>
+public class Rating
+{
+	///<summary></summary>
+	public double? CurrentRating { get; set; }
+
+	///<summary></summary>
+	public object? CommandParameter { get; set; }
```

Comment:
CommandParameter is specific for MVVM and we should not include it in Core. What is this property for?

Reply:
it will be useful when the  need for passing rating value to the viewModel comes in

Reply:
@Eel2000 the Core project should be agnostic, so it shouldn't hanve any influente of MVVM, or ReactiveExtensions or any other pattern. Please, refer to [this docs](https://github.com/CommunityToolkit/Maui/wiki/MCT-architecture#core) for more info and feel free to ping me if you need any futher assistence.